relay/nakama/events/events.go

```diff
@@ -73,24 +95,36 @@ func (eh *EventHub) Shutdown() {
 func (eh *EventHub) Dispatch(log runtime.Logger) error {
 	var err error
 	for !eh.didShutdown.Load() {
-		messageType, message, err := eh.inputConnection.ReadMessage() // will block
+		var messageType int
+		var message []byte
+		messageType, message, err = eh.inputConnection.ReadMessage() // will block
 		if err != nil {
-			err = eris.Wrap(err, "")
+			err = eris.Wrap(err, "") // This now correctly updates the outer 'err' variable
 			eh.Shutdown()
 			continue
 		}
 		if messageType != websocket.TextMessage {
 			eh.Shutdown()
 			continue
 		}
+		receivedTickResults := TickResults{}
+		err = json.Unmarshal(message, &receivedTickResults)
+		if err != nil {
+			log.Error("unable to unmarshal message into TickResults: ", err)
```

Comment:
This still attempts to emit the empty receivedTickResults. I think you want a continue here.

Reply:
Nice catch! Done